src/app.ts

```diff
@@ -38,6 +38,9 @@ const FSH_VERSION = '3.0.0-ballot';
 
 function logUnexpectedError(e: Error) {
   logger.error(`SUSHI encountered the following unexpected error: ${e.message}`);
+  if (e.stack) {
+    logger.debug(e.stack);
+  }
```

Comment:
Only the `build` command can set the log level, which means that the stack trace is only available for that command. It might be useful to be able to get the stack trace for other commands by adding the log level option for those commands.

Reply:
Ah... I hadn't thought of that. It probably makes sense to add it to the other commands too. I'm heavy on my other project today, but I can maybe look at it next week.

Reply:
3737ce4 adds support for --log-level to the other commands (`init` and `update-dependencies`). To test it, I added `throw new Error('Boo!')` in the code those commands invoke and confirmed that without `-l debug` there is no stack trace, but with it, there is.